Automattic/wp-calypso
# Plans 2023 : UI only fix for Ecommerce plan name and logo for business plan

#### Proposed Changes

* Front end only plan name update to the ecommerce plan
* Added cloud logo to business plan

<img width="1420" alt="image" src="https://user-images.githubusercontent.com/3422709/216038068-f51e8fa4-0f65-4a05-b6b5-978620834991.png">

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/3422709/216038465-3b6c454c-e9fc-4f93-a3fc-56fc3043b12a.png">

![image](https://user-images.githubusercontent.com/3422709/216038570-ad533379-8088-4373-a663-67a573ad0253.png)

#### Testing Instructions



-  Go to /start/plans?flags=onboarding/2023-pricing-grid
- Make sure the logo is updated and the eCommerce plan is changed to Commerce at all places on the page

#### Pre-merge Checklist





Related to #
Fixes Automattic/martech#1452
Fixes Automattic/martech#1395
